src/main/java/mat/client/cqlworkspace/definitions/CQLDefinitionsView.java

```diff
@@ -254,7 +257,11 @@ public void setWidgetReadOnly(boolean isEditable) {
 		getDefineButtonBar().getInsertButton().setEnabled(isEditable);
 		getDefineButtonBar().getTimingExpButton().setEnabled(isEditable);
 		getDefineButtonBar().getEraseButton().setEnabled(isEditable);
-		getDefineButtonBar().getExpressionBuilderButton().setEnabled(isEditable);
+		if (isSelectedObjectFhir()) {

```

Comment:
does it make sense to use a one-liner instead of if-else block?

getDefineButtonBar().getExpressionBuilderButton().setEnabled(isEditable && !isSelectedObjectFhir.getAsBoolean());

Reply:
Yup It makes sense. Will update it.